.travis.yml

```diff
@@ -0,0 +1,20 @@
+env:
+  - PORTAGE=2.2.15 CATEGORY=[a-c]
+  - PORTAGE=2.2.15 CATEGORY=dev-[a-o]
+  - PORTAGE=2.2.15 CATEGORY=dev-[p-z]
+  - PORTAGE=2.2.15 CATEGORY=[e-z]
```

Comment:
`/home/travis/build.sh: line 41: pushd: header.txt: Not a directory`
;D

And you failed to die properly, so it started full-tree scan and suicided after 50 mins.